StartEmailValidationProcess/orchestrators.ts

```diff
@@ -0,0 +1,45 @@
+import * as crypto from "crypto";
+import * as df from "durable-functions";
+import { DurableOrchestrationClient } from "durable-functions/lib/src/durableorchestrationclient";
+
+import { toError } from "fp-ts/lib/Either";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+import { PromiseType } from "italia-ts-commons/lib/types";
+import { EmailAddress } from "../generated/backend/EmailAddress";
+
+const hashCreator = (value: string) =>
+  crypto
+    .createHash("sha256")
+    .update(value)
+    .digest("hex");
+
+/**
+ * The identifier for EmailValidationProcessOrchestrator
+ * @param fiscalCode the id of the requesting user
+ * @param email the user's email
+ */
+export const makeStartEmailValidationProcessOrchestratorId = (
+  fiscalCode: FiscalCode,
+  email: EmailAddress
+) => hashCreator(`${fiscalCode}-${email}`);
```

Comment:
what about adding the day (dd/mm/yyyy) to the hash ? in this way if the orchestrator stucks the user can retry the day after cc @balanza

Reply:
The idea is good, I'm just a bit a little reluctant in introducing time assumptions without make them parametrisable.
We can have troubles testing the feature if we hardcode such behaviour.

Reply:
you can pass a parameter with the date having a default value of Date.now() if you're worried about testing.